argoverse/map_representation/map_api.py

```diff
@@ -379,6 +379,20 @@ def get_ground_height_at_xy(self, point_cloud: np.ndarray, city_name: str) -> np
         npyimage_coords = npyimage_coords.astype(np.int64)
 
         # index in at (x,y) locations, which are (y,x) in the image
+        max_x = np.max(npyimage_coords[:, 1])
+        max_y = np.max(npyimage_coords[:, 0])
+
+        height_x, height_y = np.shape(ground_height_mat)
+
+        assert np.all(npyimage_coords[:, 1] > 0) and np.all(npyimage_coords[:, 0] > 0), "Invalid coordinates:"
```

Comment:
[nit] Can we print a more informative error message? eg. "Invalid coordinates because of null coordinates" or something